parameter_files/fates_params_default.cdl

```diff
@@ -769,7 +769,7 @@ data:
  fates_history_sizeclass_bin_edges = 0, 5, 10, 15, 20, 30, 40, 50, 60, 70,
     80, 90, 100 ;
 
- fates_hydr_htftype_node = 1, 1, 1, 1 ;
+ fates_hydr_htftype_node = 2, 2, 2, 2 ;
```

Comment:
@JunyanDing do you recommend that most people use type 2, or is this just for your runs?  If we do switch the default, I'd like perspectives from  @pnlfang @xuchongang and others.

Reply:
This is just my preference. We can keep the default to be 1

Reply:
Reverted.